src/browser/vdiff.js

```diff
@@ -19,6 +19,13 @@ mocha.setup({
 });
 /* eslint-enable */
 
+function findRealSize(elem) {
+	if (!elem.shadowRoot) return elem;
+	const nestedSizedElem = elem.shadowRoot.querySelector('.vdiff-size');
```

Comment:
Alternate options for the CSS class: `vdiff-rect-elem`, `vdiff-rect`, `vdiff-container`

Reply:
Yeah I had `vdiff-rect` and then I didn't love it, but I don't love `vdiff-size` either so switched back to that!

Reply:
Throwing another option out there: `vdiff-boundary`

Reply:
`vdiff-area`? `vdiff-box`? `vdiff-frame`? `vdiff-actual`?

Reply:
`vdiff-target`?